mm2src/adex_cli/src/rpc_data.rs

```diff
@@ -0,0 +1,71 @@
+//! Contains rpc data layer structures that are not ready to become a part of the mm2_rpc::data module
+//!
+//! *Note: it's expected that the following data types will be moved to mm2_rpc::data when mm2 is refactored to be able to handle them*
+//!
+
+use mm2_rpc::data::legacy::{ElectrumProtocol, GasStationPricePolicy, UtxoMergeParams};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(tag = "method", rename_all = "lowercase")]
+pub(crate) enum ActivationRequest {
```

Comment:
Shouldn't this be in `mm2_rpc/src/data/legacy/activation.rs`

Reply:
No, this shouldn't cause they are [only used in the adex-cli](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1912#discussion_r1261494793)

But in [further implementation I've already made it](https://github.com/KomodoPlatform/komodo-defi-framework/blob/1799-swap-commands/mm2src/adex_cli/src/rpc_data/activation.rs#L1-L71) like you suggest here

Reply:
ooh this is in `adex-cli` path, sorry that I forgot about this. Will resolve these comments.